src/traits.rs

```diff
@@ -1522,6 +1539,17 @@ pub trait ComponentExt: Debug {
     ///     println!("{}", component.label());
     /// }
     /// ```
+    ///
+    /// ## Linux hwmon label formating
+    ///
+    /// It may change. Inspired by `sensors` own formating.
+    ///
+    /// | name | label | device_model | id_sensor | Computed label by `sysinfo`                |
+    /// |:----:|:-----:|:------------:|:----------|:-----------------------------------------:|
+    /// | ✓    | ✓    | ✓            | ✓        | `"{name} {label} {device_model} temp{id}"` |
+    /// | ✓    | ✓    | ✗            | ✓        | `"{name} {label} {id}"`                    |
+    /// | ✓    | ✗    | ✓            | ✓        | `"{name} {device_model}"`                  |
+    /// | ✓    | ✗    | ✗            | ✓        | `"{name} temp{id}"`                        |
```

Comment:
```suggestion
    /// ## Linux
    ///
    /// Since components' information is retrived thanks to `hwmon`, the labels are generated as
    /// follows (it may change and it was inspired by `sensors` own formatting):
    ///
    /// | name | label | device_model | id_sensor | Computed label by `sysinfo` |
    /// |---------|--------|------------|----------|----------------------|
    /// | ✓    | ✓    | ✓  | ✓ | `"{name} {label} {device_model} temp{id}"` |
    /// | ✓    | ✓    | ✗  | ✓ | `"{name} {label} {id}"` |
    /// | ✓    | ✗    | ✓  | ✓ | `"{name} {device_model}"` |
    /// | ✓    | ✗    | ✗  | ✓ | `"{name} temp{id}"` |
```

rustdoc se chargera de la mise en forme. ;)